solutions/control flow/rating-contest.js

```diff
@@ -0,0 +1,22 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+rating = parseInt(readLine());
```

Comment:
@prateek-mehra 

1. Please use `let` to declare variables.
2. Also, please rename the variable to `rank`.
3. Please rename the folder to `control-flow`.